### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by dipen2000**
I created the typography for my component library so please review it and give me feedback.

### CODE REVIEW

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.heading-1{
```
Comment (deekshithmd) [inline components/typography/typography.css:1]: typography code looks neat and good to merge

**Review by j24m — APPROVED** [review]
Good to merge.

**Review by anshulraheja — APPROVED** [review]
UI looks awesome. Just a suggestion, on the landing page, remove the button tag from Github and Get Started, only use the anchor tag and apply a button like CSS on it. The reason behind this is that the button is not clickable. You can check this by clicking on the sides of the button. It's the anchor tag (content) that is clickable and that takes us to the next page.

File: components/typography/typography.css (hunk 1 above)
Comment (dipen2000) [reply components/typography/typography.css:1]: Then please approve this too.

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**